contrib/pyln-client/pyln/client/lightning.py

```diff
@@ -704,7 +704,8 @@ def feerates(self, style, urgent=None, normal=None, slow=None):
     def fundchannel(self, node_id, amount, feerate=None, announce=True,
                     minconf=None, utxos=None, push_msat=None, close_to=None,
                     request_amt=None, compact_lease=None,
-                    mindepth: Optional[int] = None):
+                    mindepth: Optional[int] = None,
+                    reserve: Optional[str] = None):
```

Comment:
needs request schema and doc changes?  Clearly we don't test this by keyword, otherwise CI would barf...

Reply:
The request schema for `fundchannel` is in #5362, hence why this isn't covered by the test yet.

Reply:
👍